src/components/Label/Label.st.css

```diff
@@ -0,0 +1,37 @@
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Label, LabelStyle;
+}
+  
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: secondaryLabel;
+}
+
+:import {
+  -st-from: "../../typography.st.css";
+  -st-named: fontLight;
+}
+
+.root {
+  -st-extends: Label;
+  -st-states: appearance(string);
+}
+
+.root:appearance(T1.1) {
+  -st-mixin: LabelStyle(
+    fontFamily value(fontLight),
+    fontSize 16px,
+    lineHeight 24px,
+    color value(secondaryLabel)
+  );
+}
+
+.root:appearance(T3.1) {
```

Comment:
I see we have T1.1 & T3.1
Basically in the new API I wrote bellow, this means:
`appearance="T1.1"` --> `size="medium" secondary={true}`
`appearance="T3.1"` --> `size="small" secondary={true}`